cloudtools/scripts/aws_create_instance.py

```diff
@@ -180,15 +180,15 @@ def create_instance(name, config, region, key_name, ssh_key, instance_data,
             # without the full stack trace
             log.warn("cannot connect; instance may still be starting  %s (%s, %s) - %s,"
                      "retrying in %d sec ...", instance_data['hostname'], instance.id,
-                     instance.private_ip_address, e, FAILURE_TIMEOUT)
-            time.sleep(FAILURE_TIMEOUT)
+                     instance.private_ip_address, e, assimilate_timeout)
+            time.sleep(assimilate_timeout)
```

Comment:
For some reason this still remains 1200 when I tested. Not sure why, as in the `make_instance` call, when the process is being forked and args transmitted, it was correctly sent as the value I specified in the command line arguments (300 seconds).

Reply:
Need to do more debugging here, it still doesn't work as is or I've done something wrong in testing.